Tests/LibWeb/AccessibilityBridge/tests/test_tree_shape.py

```
@@ -0,0 +1,143 @@
+
+class TreeShapeTests(AccessibilityBridgeTestCase):
+    FIXTURE = "tree_shape.html"
+
+    def _all_text(self):
+        """Return every piece of text anywhere in the document subtree."""
+        out = []
+
+        def visit(obj, _depth):
+            try:
+                count = Atspi.Text.get_character_count(obj)
+            except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)